geps/gep-1709.md

```diff
@@ -231,6 +231,25 @@ move on to [certification](#certification) to report the results.
 [go]:https://go.dev
 [lib]:https://pkg.go.dev/sigs.k8s.io/gateway-api@v0.6.2/conformance/utils/suite
 
+### Implementation mode
+
+The certification process runs against an implementation using a specific mode
+(e.g., `standard`, `enhanced`) that is defaulted to `standard`. The final
+report will contain such a mode in the modes field. A "mode" is intended to capture
+situations where a Gateway API implementation may have different features and capabilities
+depending on how it is deployed (e.g. an implementation might be deployed with an external,
+OR internal load balancer, and have different capabilities depending on the mode chosen).
+Every certification report `modes` array contains only one element. Reports can be merged
+into one another to provide a single report that accounts for multiple modes, given the same
+Gateway API version.
```

Comment:
One thing that's not clear from this is where these modes are defined - if they are definable by each implementation, then we should say that explicitly here. (It would make sense to me that each implementation could define their own set of `mode`s.)

Reply:
It should be addressed now (see the other comment).